commentListener.py

```diff
@@ -0,0 +1,135 @@
+import math
+from coral.metrics.readability.utils import *
+from coral.metrics.readability import Readability
+import pika
+import json
+import os
+import statsd
+import sys
+
+#
+#  Usage:
+#     python commentListener.py
+#
+#  Subscribes to Coral RabbitMQ messages, filters for comments,
+#  validates messages, and calculates metrics on the text.
+#
+
+# Constants
+COMMENT_EVENT = 'comment_import'
+
+# Get some standard coral environment vars
+try:
+    rmq_user = os.environ['RABBITMQ_DEFAULT_USER']
+    rmq_pass = os.environ['RABBITMQ_DEFAULT_PASS']
+    rmq_host = os.environ['RABBITMQ_HOST']
+    rmq_port = int(os.environ['RABBITMQ_PORT'])
+    rmq_exchange = os.environ['AMQP_EXCHANGE']
+    statsd_host = os.environ['STATSD_HOST']
+    statsd_port = int(os.environ['STATSD_PORT'])
+
+    # For mongo integration
+    #mongo_db = os.environ['DBNAME']
+    #mongo_host = os.environ['MONGODB_HOST']
+except Exception as err:
+    print("ERROR: Environment variable not found. " + err, file=sys.stderr)
+    exit()
```

Comment:
you should send a non-zero exit, like `exit(1)`, that way if you include this as part of some other tooling, they can infer something wrong happened by the exit code.